### PR TITLE
libvirt 2.0.0

### PR DESCRIPTION
**Pull request by justinclift**

### CODE REVIEW

**Comment by justinclift** [comment]
Note, this is an updated replacement for PR #2691.

File: Formula/libvirt.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Libvirt < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnutls"
+  depends_on "curl"
```
Comment (DomT4) [inline Formula/libvirt.rb]: Which version of `curl` does it require? I'd be surprised if the version of cURL on El Cap for example wasn't new enough.

File: Formula/libvirt.rb [hunk 2]
```diff
@@ -62,4 +67,26 @@ def install
       end
     end
   end
+
+  test do
+    output = shell_output("#{bin}/virsh -v", 0)
```
Comment (DomT4) [inline Formula/libvirt.rb]: The `, 0` bit is assumed with `shell_output` unless it's told otherwise. Doesn't need stating explicitly.

**Comment by DomT4** [comment]
CI failure was a false fail. Jenkins had a panic. I picked up a couple of issue outside of that but we look pretty good otherwise.

File: Formula/libvirt.rb (hunk 1 above)
Comment (justinclift) [inline Formula/libvirt.rb]: Any version seems to work.  It's just that the default curl doesn't seem to be found for some unknown reason (I didn't investigate), whereas a Homebrew provided one does.

File: Formula/libvirt.rb (hunk 2 above)
Comment (justinclift) [inline Formula/libvirt.rb]: No worries.  Will push a revised commit in a sec.

File: Formula/libvirt.rb (hunk 1 above)
Comment (DomT4) [inline Formula/libvirt.rb]: Interesting. Can you provide the failure logs? Wondering if that's something we can fix fairly easily.

Comment (justinclift) [inline Formula/libvirt.rb]: Sure, give me a few minutes to run through it again. :smile:

Comment (justinclift) [inline Formula/libvirt.rb]: ... and now the failure isn't occurring.

I have no idea what was causing it the other day, but curl doesn't seem to be needed after all. 😵 

Will push a new commit in a sec.

**Comment by justinclift** [comment]
@DomT4 Sorry for the noise with curl.  No idea what happened there. 😦

**Comment by DomT4** [comment]
No worries at all! These things happen. Appreciate you double checking to spare us the unnecessary dependency.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/55faf289b26bf124f6a3a700a39b60b0a1ccf7a1. Thank you for another contribution to Homebrew @justinclift; we appreciate it! :smiley_cat:

**Comment by justinclift** [comment]
Thanks @DomT4. :smile:

**Pull request closed without merging**